scripts/access_db.py

```diff
@@ -81,6 +95,30 @@ def delete(self, args):
                 self.db.history.delete_many(query)
                 self.db.configuration.delete_many(query)
 
+    def download(self, args):
+        for experiment_id in args.experiment_id:
+            data, environment_config = data_from_database(
+                experiment_id, self.db)
+            downloaded = {
+                'data': data,
+                'environment_config': environment_config,
+                'experiment_id': experiment_id,
+            }
+            serialized = serialize_value(downloaded)
+            with open('{}.json'.format(experiment_id), 'w') as f:
+                json.dump(serialized, f)
+
+    def upload(self, args):
+        # Initialize database, including with indexes
+        for filepath in args.json_file:
+            with open(filepath, 'r') as f:
+                downloaded = json.load(f)
+            data = downloaded['data']
+            environment_config = downloaded['environment_config']
+            experiment_id = downloaded['experiment_id']
```

Comment:
How does the ```experiment_id``` get into the db with this?

Reply:
Thanks for pointing this out. We don't actually use the experiment ID that shows up in the code. The ID is already part of the downloaded experiment data because each document has an `experiment_id` key:

![Screen Shot 2020-09-30 at 09 54 40](https://user-images.githubusercontent.com/19878639/94694633-0d6d8a80-0303-11eb-86cd-5c62dee9e0e8.png)

This means we don't really need to handle the experiment ID separately. I'll fix that

Reply:
Done in 2258132725ecc15fa9e30be448feab90ec4c8a42

Reply:
@U8NWXD -- We do use the ```experiment_id``` to query the DB. If it is not included in this upload, I'm not sure we'll be able to find the experiment. For example, we use ```experiment_id``` in ```vivarium-scripts```.

Reply:
@eagmon I'm not sure I understand. Yes, we need the experiment ID, but the ID is already in every object we download from the database, so we don't need to store a separate copy here. More concretely, the experiment ID is already in `environment_config` and `data` in the code this comment references.

Reply:
Ok, I see now. Thanks @U8NWXD!